SampleApps/SPMTest/SPMTest/ViewController.swift

```diff
@@ -9,6 +9,7 @@ import BraintreeThreeDSecure
 import BraintreeUnionPay
 import BraintreeVenmo
 import PayPalDataCollector
+import KountDataCollectorWrapper
```

Comment:
Looking at the SPM vs Carthage import statements, I think it would be good for them to be aligned